docs/Withdrawals.md

```diff
@@ -0,0 +1,21 @@
+---
+id: Withdrawals
+title: Withdrawing funds from Arbitrum
+sidebar_label: Withdrawals
+---
+
+# Withdrawing funds from Arbitrum
+
+As discussed in the section on [finality](Finality.md), because Arbitrum's execution happens optimistically, the Ethereum blockchain cannot immediately confirm the correct state and must wait for the challenge window to expire (or until all challenges are resolved). 
+
+When it comes to execution within the Rollup this does not pose any problem or add any delay. The [Arbitrum Rollup protocol](Rollup_Protocol.md) manages a tree of assertions, and allows validators to pipeline execution by continuing to build the tree even before all nodes are confirmed. This means that an honest validator can continue to advance the state of the machine with confidence (and the ability to enforce) that eventually Ethereum will recognize the honest branch as the correct and valid one. And although it will take some time for Ethereum to recognize which branch is correct, anyone that is validating the chain will immediately know this.
```

Comment:
At end of this paragraph, suggest saying that anyone that is validating the chain will immediately know which branch is correct and therefore which branch will eventually be accepted by the protocol.